src/Oci8/Oci8Connection.php

```diff
@@ -117,6 +117,18 @@ public function getSchemaBuilder(): SchemaBuilder
         return new SchemaBuilder($this);
     }
 
+    /**
+     * Get the server version for the connection.
+     */
+    public function getServerVersion(): string
+    {
+        $versionQuery = $this->getPdo()->prepare('SELECT BANNER FROM v$version');
+        $versionQuery->execute();
+        $version = $versionQuery->fetch(PDO::FETCH_OBJ);
+
+        return $version->banner_full;
```

Comment:
Looks like wrong key was used.

```suggestion
        return $version->banner;
```